src/swtpm_setup/swtpm.c

```diff
@@ -67,16 +67,19 @@ static int swtpm_start(struct swtpm *self)
     gboolean success;
     GError *error = NULL;
     unsigned ctr;
+    int pidfile_fd;
+    int ret = 1;
 
-    self->pidfile = g_strjoin(G_DIR_SEPARATOR_S,
-                              self->state_path, ".swtpm_setup.pidfile", NULL);
-    pidfile_file = g_strdup_printf("file=%s", self->pidfile);
+    g_free(self->pidfile);
```

Comment:
I don't think this should be necessary since we aren't trying to re-start the program in case it fails, or did we have a memory leak here before?